### ISSUE TITLE
Different models and different image sizes can get the same epsilon

### ISSUE THREAD

**Issue by oneheartforone** [pytorch/opacus#583]
Hi, 
When I used the official cifar10 classification example, I found that using different models (such as conv, resnet18, or even a linear layer) will get the same epsilon without changing the parameters. why does this problem happen？

After that, I tried different image input sizes and found that when other conditions remained unchanged, epsilon was still the same.

Similar results can be obtained when sigma is fixed.

## Environment
 - PyTorch Version (e.g., 1.13.1):
 - Win:
 - conda
 - Python version: 3.9
 - CUDA/cuDNN version:  11.6.1
 - GPU models and configuration: 4080 16G

**Comment by alexandresablayrolles** [comment]
This is because the privacy budget (epsilon) only depends on three factors: the number of steps (or epochs), the batch size and the level of noise (sigma). In particular, the budget does not depend on the architecture or the loss function or the optimizer.
